dcc-import-go/pom.xml

```diff
@@ -84,6 +84,7 @@ ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF S
       <groupId>org.slf4j</groupId>
       <artifactId>log4j-over-slf4j</artifactId>
       <version>${slf4j.version}</version>
+      <scope>test</scope>
```

Comment:
just curious why you reduce this to test scope?

Reply:
Not my change. This was a change made by the commits from the branch